hera_cal/reflections.py

```diff
@@ -885,6 +885,7 @@ def interp_u(self, umodes, times, full_times=None, uflags=None, keys=None, overw
                 # shift by median and normalize by MAD
                 ymed = np.median(y, axis=0, keepdims=True)
                 ymad = np.median(np.abs(y - ymed), axis=0, keepdims=True) * 1.4826
+                ymad[np.isclose(ymad, 0.0)] = 1.0
```

Comment:
This seems a bit dangerous. Perhaps add a comment or documentation? Are you accounting for it in some way?

Reply:
its was done to fix a rare but catastrophic failure that the ymad == 0, in which case you get a division by zero. its actually quite safe to do, b/c in this case the data itself is just a constant